index.html

```diff
@@ -364,25 +347,25 @@ <h3>StatusList2021Entry</h3>
 The <code>statusListCredential</code> property MUST be a URL to a
 <a>verifiable credential</a>. When the URL is dereferenced, the resulting
 <a>verifiable credential</a> MUST have <code>type</code> property that
-includes the <code>StatusList2021Credential</code> value.
+includes the <code>StatusListCredential</code> value.
               </td>
             </tr>
           </tbody>
         </table>
 
-        <pre class="example nohighlight" title="Example StatusList2021Credential">
+        <pre class="example nohighlight" title="Example StatusListCredential">
 {
   "@context": [
-    "https://www.w3.org/2018/credentials/v1",
-    "https://w3id.org/vc/status-list/2021/v1"
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/status-list/v2"
```

Comment:
```suggestion
```

Reply:
Not needed, assuming the v2 context continues to bundle status list terms... also keeps the integrity check minimized to remove it.